docs/plugins/trello-github-integ_plugin.md

```diff
@@ -11,6 +11,9 @@ _This plugin depends on the following two environment variables:_
 
 Set the values accordingly before using this plugin.
 
+## 3 Tips:
+_Trello board description managed by DevStream, don't modify it_
```

Comment:
Trello board description “is” managed by DevStream, please don't modify it.

Missing "is"; add "please" to be more polite; and missing period at the end.